.circleci/config.yml

```diff
@@ -38,22 +38,33 @@ jobs:
             ensure-version-bumped.sh
             prohibit-tab-character.sh
             prohibit-trailing-whitespace.sh
+
       - run:
           name: Run tests against Python 3.6 and git 1.8.0 (without checking coverage)
-          command: PYTHON_VERSION=3.6 GIT_VERSION=1.8.0  bash ci/tox/ci-run.sh  # Earliest versions of python/git supported by git-machete
+          command: PYTHON_VERSION=3.6 GIT_VERSION=1.8.0  bash ci/run-dockerized.sh tox # Earliest versions of python/git supported by git-machete
       - run:
           name: Run tests against Python 3.7 and git 2.7.6 (without checking coverage)
-          command: PYTHON_VERSION=3.7 GIT_VERSION=2.7.6  bash ci/tox/ci-run.sh
+          command: PYTHON_VERSION=3.7 GIT_VERSION=2.7.6  bash ci/run-dockerized.sh tox
       - run:
           name: Run tests against Python 3.8 and git 2.25.0 (without checking coverage)
```

Comment:
Nit: while you're here, remove `(without checking coverage)`, feels too verbose (leave just `(with checking coverage)` below)